### PR TITLE
Destination and Budget details

### PR DESCRIPTION
**Pull request by bertacume**

### CODE REVIEW

File: thesis-fe/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "apollo-link-http": "^1.5.7",
     "emotion": "^9.2.12",
     "graphql": "^14.0.2",
+    "jquery": "^3.3.1",
```
Comment (ArtyBMPS) [inline thesis-fe/package.json]: jQuery?

Comment (bertacume) [reply thesis-fe/package.json]: it was needed for a component i was using from a library, but actually i removed that component so i should remove it from the json file. Doing it now

**Review by ArtyBMPS — APPROVED** [review]

**Review by ArtyBMPS — CHANGES_REQUESTED** [review]
Please remove the jQuery dependency

**Review by ArtyBMPS — APPROVED** [review]
Ok

**Pull request merged**